packages/dd-trace/src/appsec/iast/taint-tracking/rewriter.js

```diff
@@ -181,16 +183,17 @@ function isEsmConfigured () {
   const hasLoaderArg = isFlagPresent('--loader') || isFlagPresent('--experimental-loader')
   if (hasLoaderArg) return true
 
-  const initializeLoaded = Object.keys(require.cache).find(file => file.includes('import-in-the-middle/hook.js'))
-  return !!initializeLoaded
+  // Fast path for common case when enabled
+  if (require.cache[`${process.cwd()}/node_modules/import-in-the-middle/hook.js`]) {
```

Comment:
This would be more useful for serverless, but in serverless the tracer is installed elsewhere. On servers, I don't think the tiny gain justifies the added code.

Reply:
This would be also problematic for us, in development, when I am trying the tracer locally in a custom app, the tracer is in other directory with their own `node_modules`, not in `process.cwd()`.

Reply:
@duncanista Do you know what the path would be in Lambda so that it can also benefit from this change?